Dockerfile

```diff
@@ -23,7 +24,7 @@ COPY ./build .
 
 WORKDIR /
 
-RUN TRAVIS_BUILD_NUMBER=$TRAVIS_BUILD_NUMBER MYGET_API_KEY=$MYGET_API_KEY dotnet run --project build/build.csproj
+RUN TRAVIS_OS_NAME=$TRAVIS_OS_NAME TRAVIS_BUILD_NUMBER=$TRAVIS_BUILD_NUMBER MYGET_API_KEY=$MYGET_API_KEY dotnet run --project build/build.csproj
```

Comment:
Likewise, any _remaining_ args need to be passed through to Bullseye here, otherwise you cannot pass any args to Bullseye from `build.sh`.